### PR TITLE
feat(app): add clone run to recent run protocol card

### PR DESCRIPTION
**Pull request by koji**
# Overview
Currently when tapping a RecentRunProtocolCard, the app move to protocol details but this doesn't align with Desktop app. When tapping a card, the app clone the prior run, including any labware position check offset data. 
close RCORE-740

# Test Plan
- send an protocol to an OT-3 (if a robot doesn't have)
- run a protocol once
- go to `All Protocols` and tap a RecentProtocolRunCard
The app shows Setup protocol screen and clone a new run from the previous run

# Changelog
- modify handleCardClick in RecentRunProtocolCard
- update ProtocolSetup.tsx (update buttons and remove unused items)

# Review requests

# Risk assessment
low

### CODE REVIEW

**Comment by koji** [comment]
- [x] add test case
- [x] Robot Dashboard header position <- This has been fixed by https://github.com/Opentrons/opentrons/pull/12554

File: app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx [hunk 1]
```diff
@@ -315,7 +322,7 @@ function PrepareToRun({
             <PlayButton disabled={!isReadyToRun} onPlay={onPlay} />
           </Flex>
         </Flex>
-        <Flex gridGap={SPACING.spacing4}>
+        {/* <Flex gridGap={SPACING.spacing4}>
```
Comment (jerader) [inline app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx]: can you add a comment to explain why this is commented out?

**Review by jerader — APPROVED** [review]
lgtm!

File: app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx (hunk 1 above)
Comment (koji) [reply app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx]: oh, let me check.

Comment (koji) [reply app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx]: Eric updated ProtocolSetup to align the UI with the hi-fi.
I will merge edge into this branch and solve the conflicts so this comment will be removed.

**Review by b-cooper — APPROVED** [review]
💯 looks good to merge

**Comment by koji** [comment]
- [ ] ask about target path when cancel a run in ProtocolSetup screen app&ui<> design

**Pull request merged**